src/pyfaf/actions/repomod.py

```diff
@@ -98,6 +98,6 @@ def tweak_cmdline_parser(self, parser):
 
         group = parser.add_mutually_exclusive_group()
         group.add_argument("--gpgcheck", action="store_true",
-                           help="enable gpg check for this repository")
+                           help="enable GPG check for this repository")
         group.add_argument("--nogpgcheck", action="store_true",
-                           help="disable gpg check for this repository")
+                           help="disable GPG check for this repository")
```

Comment:
It's meant to modify an existing property: `--nogpgcheck` to disable GPG check for a repo that has it enabled and `--gpgcheck` for the opposite.  
  
If `--nogpgcheck` were the default, then disabling  GPG check for a repo that has it enabled would require running `faf repomod` with no arguments at all, i.e. don't modify anything and set GPG check to the default value. Which is neither very intuitive (running the program without arguments) nor consistent behaviour (should the thing be left untouched, or set to default?).

Reply:
Oh, I understand now. If it's actually an action modifying a property, it seems more prudent to have it in a form like `--gpg-check=yes/no` to enable/disable the check explicitly, thus rendering the juggling of mutually exclusive argument void. In the web UI, this could be a dropdown with options such as "Do not change", "Enable" and "Disable".

Reply:
Yeah, that could be done. @mkutlak, would reworking the `--(no)gpgcheck` arg to `faf repomod` like this break anything? Up until now, I was trying not to change the behaviour of the CLI.

Reply:
@michalfabik No

Reply:
Done